### PR TITLE
html

### PR DESCRIPTION
**Pull request by boone824**
fixed html to flex container, add css.

### CODE REVIEW

File: stylesheet/styles.css [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+}
+body {
+  margin: 0px;
+  padding: 0px;
+}
+/* Header */
+header {
+  background-color: black;
+  color: white;
+  background-image: url('https://boone824.github.io/adopt-a-dog-winter2020/images/header-image.jpg');
```
Comment (jimmynono) [inline stylesheet/styles.css]: you should call a relative path to this file `background-image: url('./images.header-image.jpg');`

File: stylesheet/styles.css [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+}
+#brand-container {
```
Comment (jimmynono) [inline stylesheet/styles.css]: keep all id's for js reasons, just make a class, it's ok if you only use it once.

File: stylesheet/styles.css [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+  padding: 5px 20px;
+  color: white;
+  margin-right: 2em;
+  width: 120px;
+  height: 25px;
+}
+#checkout-btn:hover {
+  background-color: black;
+  cursor: pointer;
+  transition: all 0.5s ease;
```
Comment (jimmynono) [inline stylesheet/styles.css:43]: nice transition! lovely

File: stylesheet/styles.css [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+  color: red;
+  border: 1px red solid;
```
Comment (jimmynono) [inline stylesheet/styles.css:45]: proper syntax is `border: 1px solid red;`

File: stylesheet/styles.css [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+  height: 3rem;
+  margin-top: 12rem;
+}
+nav ul, {
+  margin: 0 auto;
+  width: 70em;
+
+}
+main {
+  width: 80%;
```
Comment (jimmynono) [inline stylesheet/styles.css]: I'd say width should be 100%, but set a max width so it doesn't go too wide on desktop

File: stylesheet/styles.css [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+  font-size: 16px;
+  color: white;
+  line-height: 2em;
+}
+@media screen and (max-width:1024px) {
+  #content-container, #adopt-container,
+  header, .content text, .ft-nav {
+    width: 100%;
+  }
+  .adopt-item-1 {
```
Comment (jimmynono) [inline stylesheet/styles.css]: we'll talk about media queries later, but I'd aim to have 4 wide on desktop, three wide on tablet and 1 wide on mobile.

**Comment by boone824** [comment]
added media q assignment

**Comment by boone824** [comment]
fix Header and added gradient

**Pull request closed without merging**